src/bentoml/_internal/io_descriptors/base.py

```diff
@@ -47,6 +48,26 @@ def from_spec(spec: dict[str, t.Any]) -> IODescriptor[t.Any]:
     return IO_DESCRIPTOR_REGISTRY[spec["id"]].from_spec(spec)
 
 
+def append_to_last_docstring(docstring: str):
+    def wrapper(f: F) -> F:
+        f.__doc__ += docstring
+        return f
+
+    return wrapper
+
+
+FROM_SAMPLE_NOTES = """\
+
+        .. note::
+
+            ``from_sample`` does not take positional arguments.
```

Comment:
What does this mean?

Reply:
This means `from_sample` will only accept kwargs arguments.